app/frontend/src/features/communities/PagePage.tsx

```diff
@@ -0,0 +1,61 @@
+import PageTitle from "../../components/PageTitle";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { useAppDispatch } from "../../store";
+import { Page } from "../../pb/pages_pb"
+import { service } from "../../service";
+import Alert from "../../components/Alert";
+import CircularProgress from "../../components/CircularProgress";
+import TextBody from "../../components/TextBody";
+import { useHistory } from "react-router-dom";
+import { pageRoute } from "../../AppRoutes"
+import Markdown from "../../components/Markdown";
+
+export default function PagePage() {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const [page, setPage] = useState(null as Page.AsObject | null);
```

Comment:
Explicit type generics are preferred to type assertions.
```suggestion
  const [page, setPage] = useState<Page.AsObject | null>(null);
```